config-model/src/main/java/com/yahoo/vespa/model/admin/monitoring/VespaMetricSet.java

```diff
@@ -131,6 +131,11 @@ private static Set<Metric> getConfigServerMetrics() {
         addMetric(metrics, ConfigServerMetrics.ZK_CONNECTIONS.last());
         addMetric(metrics, ConfigServerMetrics.ZK_OUTSTANDING_REQUESTS.last());
 
+        addMetric(metrics, ConfigServerMetrics.CLUSTER_COST.last());
+        addMetric(metrics, ConfigServerMetrics.CLUSTER_LOAD_IDEAL_CPU.last());
+        addMetric(metrics, ConfigServerMetrics.CLUSTER_LOAD_IDEAL_MEMORY.last());
+        addMetric(metrics, ConfigServerMetrics.CLUSTER_LOAD_IDEAL_DISK.last());
```

Comment:
The node-repo metrics are not reported via metrics-proxy, so there is no need to add them here.

Reply:
These sets are only allow-lists for the metrics-proxy, so adding them is a NOP. But I guess they could be processed by the proxy some time in the future.